cli/running/running.go

```diff
@@ -581,6 +587,29 @@ func renderInstCtxMembers(instance *InstanceCtx) error {
 	return nil
 }
 
+// GetClusterConfigPath returns a cluster config path for the application.
+// If mustExist flag is set and config is not found, ErrNotExists error is returned,
+// default config filepath is returned otherwise.
+func GetClusterConfigPath(cliOpts *config.CliOpts,
+	ttConfigDir, appName string, mustExist bool) (string, error) {
+	instEnabledPath := cliOpts.Env.InstancesEnabled
+	if instEnabledPath == "." {
+		instEnabledPath = ttConfigDir
+	}
+	configPath := filepath.Join(instEnabledPath, appName, clusterConfigDefaultFileName)
```

Comment:
This is what I was talking about: duplicating logic can cause errors. If instances enabled is . and config directory is an application, `config.y[a]ml` is located in instances_enabled directly, not in application sub-dir.